internal/cloud/realm/dependencies_test.go

```diff
@@ -106,5 +114,13 @@ func parseZipArchive(t *testing.T, file *os.File) map[string]string {
 	zipPkg, err := zip.NewReader(file, fileInfo.Size())
 	assert.Nil(t, err)
 
-	return parseZipPkg(t, zipPkg)
+	fileNames := make(map[string]bool, len(zipPkg.File))
```

Comment:
[naming nit] `filename`/`filenames` would be the preferred form of this `var` name within this codebase (something more general like `out` is totally fine within the scope of this helper func)